draft-packages/rating-scale-legacy/KaizenDraft/RatingScaleLegacy/types.ts

```diff
@@ -0,0 +1,8 @@
+export type ScaleValueResponse = -1 | 1 | 2 | 3 | 4 | 5
```

Comment:
This might be better named `ScaleValue`, given we don't need that extra `ScaleValueRequest` here.